runtime/autoload/dist/vim.vim

```diff
@@ -0,0 +1,30 @@
+" Vim runtime support library,
+" runs the vim9 script version or legacy script version
+" on demand (mostly for Neovim compatability)
+"
+" Maintainer:	The Vim Project <https://github.com/vim/vim>
+" Last Change:	2023 Nov 04
+
+
+" enable the zip and gzip plugin by default, if not set
+if !exists('g:zip_exec')
+  let g:zip_exec = 1
+endif
+
+if !exists('g:gzip_exec')
+  let g:gzip_exec = 1
+endif
+
+if !exists(":def")
+    function dist#vim#IsSafeExecutable(filetype, executable)
+    let cwd = getcwd()
+    return get(g:, a:filetype .. '_exec', get(g:, 'plugin_exec', 0)) &&
+          \ (fnamemodify(exepath(a:executable), ':p:h') !=# cwd
+          \ || (split($PATH, has('win32') ? ';' : ':')->index(cwd) != -1 &&
+          \  cwd != '.'))
+    endfunction
+else
+    def dist#vim#IsSafeExecutable(filetype: string, executable: string): bool
+      return dist#vim9#IsSafeExecutable(filetype, executable)
```

Comment:
This line throws E133, as `:return` is still expected to be parsed from inside a `:function` even when evaluation is being skipped over the dead `:else` branch.

The correct way to fix this would be to use `:finish` similar to what's shown in `:h vim9-mix`; I'll open an upstream PR for this.

Reply:
Thanks, I'll include it!